packages/apulse.rb

```diff
@@ -32,10 +32,13 @@ def self.build
     FileUtils.cd('build') do
       system "cmake -DCMAKE_INSTALL_PREFIX=#{CREW_PREFIX} -DCMAKE_BUILD_TYPE=Release .."
       system 'make'
-      system "echo 'pcm.plugdmix {' > asoundrc"
-      system "echo '  type plug' >> asoundrc"
-      system "echo '  slave.pcm \"dmix\"' >> asoundrc"
-      system "echo '}' >> asoundrc"
+      @asoundrc = <<~EOF
+        pcm.plugdmix {
+          type plug
+          slave.pcm "dmix"
+        }
+      EOF
+      IO.write(".asoundrc", @asoundrc)
```

Comment:
@saltedcoffii: Can we remove the binaries to get ready for the new batch?

Reply:
Fine by me! Go ahead and do anything you need.